pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -121,14 +121,17 @@ func (ctx *domainContext) publishAssignableAdapters() {
 }
 
 var debug = false
-var debugOverride bool // From command line arg
+var debugOverride bool          // From command line arg
+var hyper hypervisor.Hypervisor // Current hypervisor
 
 func Run(ps *pubsub.PubSub) {
 	handlersInit()
 	versionPtr := flag.Bool("v", false, "Version")
 	debugPtr := flag.Bool("d", false, "Debug flag")
 	curpartPtr := flag.String("c", "", "Current partition")
+	hypervisorPtr := flag.String("h", "xen", "Current hypervisor (xen, kvm, acrn, null). Default is xen")
```

Comment:
Can we move the option names `"(xen, kvm, acrn, null). Default is xen"` out of here and into some constant in the `hypervisor` package? We shouldn't have to do the detail here.

Reply:
I don't think we should though -- this is a property of a calling code (to decide what's the default, etc.) The hypervisor package itself is pretty oblivious as to what clients want the default to be (may be it will be kvm for a different client). What I do agree with, however, is making a list of hypervisors be a property of the hypervisor package. Will this work for you @deitch ?

Reply:
Sure

Reply:
Sure

Reply:
Take a look at the latest commit -- this should do it I think.